lib/hammer_cli_katello/content_override.rb

```diff
@@ -39,7 +39,7 @@ def request_params
           override = { 'content_label' => option_content_label }
           override['value'] = option_value if option_value
           override['remove'] = true if option_remove?
-          override['name'] = option_name
+          override['override_name'] = option_override_name
```

Comment:
This change is unnecessary imo. Cant you just do
```
override['name'] = option_override_name
```
and get away with it. So you keep the ```--override-name``` option but not change the API